ui/apps/dashboard/src/app/(dashboard)/env/[environmentSlug]/apps/useApps.ts

```diff
@@ -7,6 +7,7 @@ const query = graphql(`
       apps {
         id
         externalID
+        functionCount
```

Comment:
I don't see this in the new resolver changes in: https://github.com/inngest/monorepo/pull/2162

Is that in another PR?

Reply:
It'd probably help if I committed and pushed that change 😭. It's there now